checklists/alz_checklist.en.json

```diff
@@ -324,7 +321,7 @@
         {
             "category": "Identity and Access Management",
             "subcategory": "Identity",
-            "text": "Where required, use Microsoft Entra ID Application Proxy to give remote users secure and authenticated access to internal applications (hosted in the cloud or on-premises).",
```

Comment:
We should define a policy for uniform usage of trailing "." at the end of "text" field. In some cases is used, in others it is not. For some entries, it is not applied consistently.